src/SprykerMiddleware/Zed/Process/Business/Renderer/JsonRenderer.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Business\Renderer;
+
+class JsonRenderer implements RendererInterface
+{
+    /**
+     * @param mixed $data
+     *
+     * @return string
+     */
+    public function render($data)
+    {
+        return json_encode($data);
```

Comment:
Should be replaced with bridge to spryker/util-encoding?

Reply:
Yes please.